src/corevideo.cs

```diff
@@ -202,23 +230,130 @@ interface CVImageBuffer : CVBuffer {
 		[Field ("kCVImageBufferColorPrimariesKey")]
 		NSString ColorPrimariesKey { get; }
 
-		[Field ("kCVImageBufferColorPrimaries_ITU_R_709_2")]
+		[Static]
+		[Wrap ("CVImageBufferColorPrimaries.ItuR709_2.GetConstant ()")]
 		NSString ColorPrimaries_ITU_R_709_2 { get; }
 
-		[Field ("kCVImageBufferColorPrimaries_EBU_3213")]
+		[Static]
+		[Wrap ("CVImageBufferColorPrimaries.Ebu3213.GetConstant ()")]
 		NSString ColorPrimaries_EBU_3213 { get; }
 
-		[Field ("kCVImageBufferColorPrimaries_SMPTE_C")]
+		[Static]
+		[Wrap ("CVImageBufferColorPrimaries.SmpteC.GetConstant ()")]
 		NSString ColorPrimaries_SMPTE_C { get; }
 
+		[Static]
+		[Wrap ("CVImageBufferColorPrimaries.P22.GetConstant ()")]
 		[iOS (6,0), Mac (10,8)]
-		[Field ("kCVImageBufferColorPrimaries_P22")]
 		NSString ColorPrimaries_P22 { get; }
 
 		[iOS (8,0), Mac (10,10)]
 		[Field ("kCVImageBufferAlphaChannelIsOpaque")]
 		NSString AlphaChannelIsOpaque { get; }
-		
+
+		[iOS (11,0), Mac (10,13), TV (11,0)]
+		[Field ("kCVImageBufferMasteringDisplayColorVolumeKey")]
+		NSString MasteringDisplayColorVolumeKey { get; }
+
+		[iOS (11,0), Mac (10,13), TV (11,0)]
+		[Field ("kCVImageBufferContentLightLevelInfoKey")]
+		NSString ContentLightLevelInfoKey { get; }
+	}
+
+	[Watch (4,0)]
+	enum CVImageBufferTransferFunction {
+
+		[Field (null)]
+		Unknown = 2,
```

Comment:
Why 2?

Reply:
Oh I should put a comment there, it is documented as such, `2 (the code point for "unknown")`

Reply:
Ah, given `[Field (null)]` I thought it was a custom value you had put in.